stdlib/public/core/StringBridge.swift

```diff
@@ -17,34 +17,113 @@ import SwiftShims
 internal typealias _CocoaString = AnyObject
 
 #if _runtime(_ObjC)
+
 // Swift's String bridges NSString via this protocol and these
 // variables, allowing the core stdlib to remain decoupled from
 // Foundation.
 
+@objc internal protocol _StringSelectorHolder : _NSCopying {
```

Comment:
If this isn't being used externally, why is it `internal`?

Reply:
Fixed